cfme/tests/physical_infrastructure/ui/test_physical_switch_details.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_switch(appliance, provider, setup_provider_modscope):
+    # Get and return the first physical switch
+    physical_switches = appliance.collections.physical_switches.all(provider)
+    yield physical_switches[0]
```

Comment:
**REQUIRED**
No need to yield a value, just use `return` here.

Reply:
Done.